Applications/Terminal/Terminal.h

```diff
@@ -132,7 +138,10 @@ class Terminal final : public GFrame {
         return *m_lines[index];
     }
 
-    Line** m_lines { nullptr };
+    Vector<OwnPtr<Line>> m_lines {};
```

Comment:
No need for {} here.
```suggestion
    Vector<OwnPtr<Line>> m_lines;
```